lib/browser/chrome-devtools.js

```diff
@@ -97,22 +97,18 @@ ipcMainUtils.handle('ELECTRON_INSPECTOR_SELECT_FILE', async function (event) {
   return [path, data]
 })
 
-ipcMainUtils.handle('ELECTRON_INSPECTOR_CONFIRM', function (event, message, title) {
-  return new Promise((resolve, reject) => {
-    assertChromeDevTools(event.sender, 'window.confirm()')
-
-    if (message == null) message = ''
-    if (title == null) title = ''
-
-    const options = {
-      message: `${message}`,
-      title: `${title}`,
-      buttons: ['OK', 'Cancel'],
-      cancelId: 1
-    }
-    const window = event.sender.getOwnerBrowserWindow()
-    dialog.showMessageBox(window, options, (response) => {
-      resolve(response === 0)
-    })
-  })
+ipcMainUtils.handle('ELECTRON_INSPECTOR_CONFIRM', async function (event, message, title) {
+  assertChromeDevTools(event.sender, 'window.confirm()')
+
+  if (message == null) message = ''
+  if (title == null) title = ''
+
+  const options = {
+    message: `${message}`,
+    title: `${title}`,
```

Comment:
weird

Reply:
should prob just be 
```
    message, title
```

Reply:
true

Reply:
If we assume that message and title are already string. Which is probably fine.

Reply:
browser's `window.confirm()` stringifies the message